docs/developer/app-store/apps/stripe/supported-payment-methods.mdx

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_label: Supported payment methods
+sidebar_position: 7
+title: Stripe App Supported Payment methods
+---
+
+Saleor team ensures Stripe App works properly with selected payment methods.
+App doesn’t support every method available in Stripe. Saleor team keeps to add them as soon as possible.
+
+Current list of payment methods is:
+
+- Card
```

Comment:
We can add link to https://docs.stripe.com/payments/payment-methods/overview#cards